dbms/programs/client/Client.cpp

```diff
@@ -1099,6 +1118,8 @@ class Client : public Poco::Util::Application
     /// Also checks if query execution should be cancelled.
     void receiveResult()
     {
+        signal(SIGINT, SIG_DFL);
```

Comment:
Is it really needed?
(We block signals in InterruptListener.)

Reply:
Without it exit by double press Ctrl+c doesn`t work.

Reply:
How it can be explained?